pkg/apis/v1alpha1/awsnodetemplate.go

```diff
@@ -83,6 +83,13 @@ type AWSNodeTemplateSpec struct {
 	// DetailedMonitoring controls if detailed monitoring is enabled for instances that are launched
 	// +optional
 	DetailedMonitoring *bool `json:"detailedMonitoring,omitempty"`
+	// DedicatedHost contains the configuration to launch nodes on AWS Dedicated hosts
+	// +optional
+	Placement Placement `json:"placement,omitempty"`
```

Comment:
This should be a pointer to Placement since structs don't have zero-values, so omitempty won't work properly. 

```suggestion
	Placement *Placement `json:"placement,omitempty"`
```